README.md

```diff
@@ -20,43 +20,40 @@ composer require craftgate/craftgate
 ```
 
 ### Manual Installation
-You need to download the latest [release](https://github.com/craftgate/craftgate-php-client/releases) and copy to your project. Then, include the bootstrap file as shown below. This file will autoload all the class files into your project.
+You need to download the latest [release](https://github.com/craftgate/craftgate-php-client/releases) and copy to your project. Then, include the autoload file as shown below. This file will autoload all the class files into your project.
 
 ```php
-require_once('/path/to/craftgate-php-client/CraftgateBootstrap.php');
+require '/path/to/craftgate-php-client/autoload.php';
 ```
 
 ## Usage
 To access the Craftgate API you'll first need to obtain API credentials (e.g. an API key and a secret key). If you don't already have a Craftgate account, you can signup at [https://craftgate.io/](https://craftgate.io)
 
-Once you've obtained your API credentials, you can start using Craftgate by instantiating a `Craftgate` with your credentials.
+Once you've obtained your API credentials, you can start using Craftgate by instantiating a `Craftgate\Client` with your credentials.
 
 ```php
+use Craftgate\Client;
 
-$options = new RequestOptions();
-$options->setApiKey('<YOUR API KEY>');
-$options->setSecretKey('<YOUR SECRET KEY>');
-
-$craftgate = new Craftgate($options);
-...
-
+$client = new Client(array(
+    'apiKey' => '<YOUR API KEY>',
+    'secretKey' => '<YOUR SECRET KEY>',
+));
 ```
 
 By default the Craftgate client connects to the production API servers at `https://api.craftgate.io`. For testing purposes, please use the sandbox URL `https://sandbox-api.craftgate.io` using the .
 
 ```php
+use Craftgate\Client;
 
-$options = new RequestOptions();
-$options->setApiKey('<YOUR API KEY>');
-$options->setSecretKey('<YOUR SECRET KEY>');
-$options->setBaseUrl('https://sandbox-api.craftgate.io');
-
-$craftgate = new Craftgate($options);
-
+$client = new Client(array(
```

Comment:
Tüm clientlarda bir standartımız var ve class isimleri uyumludur. Bu yüzden eski halinin kalması bizim için önemli.

Reply:
Node client'a baktığımda `const craftgate = new Craftgate.Client` şeklinde bir kullanım gördüm, bunun PHP karşılığı `$craftgate = new \Craftgate\Client` oluyor (aslında `$client = new \Craftgate\Client`, init edilen `Client` sınıfı sonuçta). Yani `Craftgate` namespace'i altından bir çağrım gerçekleşiyor. Ben README'de örneklerin hemen üstünde `use Craftgate\Client` kullandığım için sadece `new Client` demek yeterli oluyor örneklerde. Aslında Node'da da `const Client = require('@craftgate/craftgate').Client` şeklinde bir import olabilir `index.ts`'ye bakınca. O nedenle değişiklik yapmam uygun olur diye düşünmüştüm. Zira buradaki değişken adı da yanlış kanımca (naming), `const craftgate = new Craftgate.Client` => `const client = new Craftgate.Client` şeklinde olmalı diye düşünüyorum, çünkü init `Client` sınıfı üzerinde gerçekleşiyor.

Sınıfları tekrar isimlendirebilirim dilerseniz.

Reply:
node.js tarafına bakalım. fakat bu case için bir standardımız olduğu için Client ismi yerine Craftgate kalması daha doğru bizim için. clientı kullanan kişinin Client objesini init etmesini istemeyiz okunaklı gelmiyor çünkü.

Reply:
Done: e46e321d68d5e98213886eba455a714df3aa5324